.github/workflows/sdk-release.yaml

```diff
@@ -38,6 +38,7 @@ jobs:
       APTOS_NODE_URL: http://127.0.0.1:8080/v1
       APTOS_FAUCET_URL: http://127.0.0.1:8081
       FAUCET_AUTH_TOKEN: ${{ secrets.DEVNET_TAP_AUTH_TOKEN }}
+      APTOS_INVOCATION: docker run ${{ secrets.GCP_DOCKER_ARTIFACT_REPO }}/tools:${{ inputs.GIT_SHA }} aptos"
```

Comment:
I think you've got a rogue `"` here.

Reply:
haha CI just told me that!

Reply:
Btw you might need the `--network host` flag too. But try it without it first and see what happens.